Tasks/VSMobileCenterTest/task.json

```diff
@@ -315,30 +210,177 @@
+            "groupName": "run",
+            "defaultValue": "",
+            "helpMarkDown": "String to identify what devices this test will run against.  Copy and paste this string when you define a new test run from Mobile Center Test beacon."
+        },
+        {
+            "name": "series",
+            "type": "string",
+            "label": "Test Series",
+            "defaultValue": "master",
+            "visibleRule": "enableRun = true",
+            "required": false,
+            "groupName": "run",
+            "helpMarkDown": "The series name for organizing test runs (e.g. master, production, beta)."
+        },
+        {
+            "name": "dsymDir",
+            "type": "string",
+            "label": "dSym Directory",
+            "defaultValue": "",
+            "visibleRule": "enableRun = true",
+            "required": false,
+            "groupName": "run",
+            "helpMarkDown": "Path to iOS symbol files."
+        },
+        {
+            "name": "locale",
+            "type": "pickList",
+            "label": "System Language",
+            "defaultValue": "en_US",
+            "visibleRule": "enableRun = true",
+            "required": true,
+            "groupName": "run",
+            "options": {
+                "da_DK": "Danish (Denmark)",
+                "nl_NL": "Dutch (Netherlands)",
+                "en_GB": "English (United Kingdom)",
+                "en_US": "English (United States)",
+                "fr_FR": "French (France)",
+                "de_DE": "German (Germany)",
+                "ja_JP": "Japanese (Japan)",
+                "ru_RU": "Russian (Russia)",
+                "es_MX": "Spanish (Mexico)",
+                "es_ES": "Spanish (Spain)",
+                "user": "Other"
+            },
+            "helpMarkDown": "If your language isn't displayed, select 'Other' and enter its locale below, such as en_US."
+        },
+        {
+            "name": "userDefinedLocale",
+            "type": "string",
+            "label": "Other Locale",
+            "defaultValue": "",
+            "required": false,
+            "groupName": "run",
+            "visibleRule": "enableRun = true && locale = user",
+            "helpMarkDown": "Enter any two-letter ISO-639 language code along with any two-letter ISO 3166 country code in the format [language]_[country], such as en_US."
+        },
+        {
+            "name": "loginOpts",
+            "type": "string",
+            "label": "Addtional Options for Login",
+            "defaultValue": "",
+            "required": false,
+            "groupName": "run",
+            "helpMarkDown": "Additional arguments passed to mobile-center login step.",
+            "visibleRule": "enableRun = true && credsType = inputs"
         },
         {
-            "name": "opts",
+            "name": "runOpts",
             "type": "string",
-            "label": "Addiitonal Options for Test Run",
+            "label": "Additonal Options for Run",
```

Comment:
One typo to another typo?  :D  I got my 't' and 'i' swapped.  You removed the 'i'.